third-party/realdds/include/realdds/dds-stream-server.h

```diff
@@ -3,8 +3,10 @@
 
 #pragma once
 
-#include "dds-stream-profile.h"
-#include "dds-stream-base.h"
+#include <realdds/dds-stream-profile.h>
+#include <realdds/dds-stream-base.h>
+
+#include <rsutils/json.h>

```

Comment:
In the header I think maybe you want `json_fwd.hpp`?

Reply:
Actually, it's no longer needed, right?

Reply:
Yes, for current implementation it is not needed, I will remove.